mkdocs.yml

```diff
@@ -4,14 +4,69 @@ site_url: https://emacs-lsp.github.io/lsp-mode
 nav:
   - Home: README.md
   - LSPs:
+      - Ada: lsp-ada.md
       - Apple Sourcekit: https://emacs-lsp.github.io/lsp-sourcekit
+      - Angular: lsp-angular.md
+      - Bash: lsp-bash.md
+      - C++: lsp-ccls.md
+      - C++: lsp-clangd.md
+      - C++: lsp-cquery.md
```

Comment:
Can you indicate that cquery is no longer supported?

Reply:
maybe we can remove it for the navigation?

Reply:
works for me. The emacs package is broken for a long time.